client/verta/verta/_deployment/endpoint.py

```diff
@@ -135,6 +136,9 @@ def update(self, run, strategy, wait=False, resources=None, autoscaling=None, en
         if not isinstance(strategy, _UpdateStrategy):
             raise TypeError("strategy must be an object from verta.deployment.strategies")
 
+        if autoscaling and not isinstance(autoscaling, Autoscaling):
+            raise TypeError("autoscaling must be an Autoscaling object")
```

Comment:
```suggestion
            raise TypeError("`autoscaling` must be an Autoscaling object")
```

Reply:
Fixed. Also fixed the ones for run and strategy (which I also missed :P)